src/components/EnketoForm.tsx

```diff
@@ -15,8 +15,8 @@ interface EnketoFormProps {
 
```

Comment:
It's really hard to review the code in this file because there's lots of stuff that has just been commented out - you should just delete stuff (that's what git is for, right) - and also lot's of changes to the logging which is making the diff really messy (see my other comment) - perhaps you could tidy this up a bit (and resolve my other comments) and then rerequest my review.